### PR TITLE
Bookstore styling

### PR DESCRIPTION
**Pull request by AjroudiRida**
in this feature branch, I did the following:

Add styling
match the Zeplin's design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,12 +117,10 @@ npm test
```
Comment (tresorsawasawa) [inline README.md:117]: - [ ] As required for the project, your rtyling to be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/), kindly ensure that you have updated it the match the expected result. Right now, your app has one book there is a big space at the top and with more than 2 books the `Header` is heidden.
## Image with one book
Screenshot from 2023-08-31 18-40-59

## Image with many books
Screenshot from 2023-08-31 18-40-32

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @AjroudiRida 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -117,12 +117,10 @@ npm test
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](http://ridaajroud.me/math-magicians/issues).
+Feel free to check the [issues page](http://ridaajroud.me/Bookstore/issues).
```
Comment (Alfredbis29) [inline README.md:121]: - [OPTIONAL ] Your README is informative, but I noticed repetition in the contribution section. Consider condensing this info to keep the README concise and engaging.
Screenshot from 2023-08-31 20-23-04

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @AjroudiRida 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] Check under the review

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**